2016-06-03.md

```diff
@@ -0,0 +1,34 @@
+# Improving Rust, Letting Go of Python, Open Finances, and More
+
+## Multiple language support
+
+### Ramping up Rust
+We've long been intrigued by the Rust Language, and have lately been pushing forward for a more native Rust support experience, including easy deploy from the `t2` command line tool.
+
+You can see the state of the project at the [tessel-rust](https://github.com/tessel/tessel-rust) repo. If you're testing it out, please let us know what you think! Issues, PRs, and conversation in the #rust-lang channel of the [Tessel Slack](//tessel-slack.herokuapp.com) are all welcome.
```

Comment:
As we mentioned before, I think we should remove this link. Instead, it might be easier to describe where we are in the process (have added `t2 init`, finishing up `t2 run/push` with cross compilation server and researching using `rustup` for local cross compilation. Also, looking for help porting JS to Rust. Should I write about my experience, in collaboration with @cflewis, adding Rust support with our language plugin system here @Frijol?